test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
Yes, it would be temporary if you found you needed it (make sure there are bugs filed against things that fail to resolve indicating they need to switch to `netcoreapp`)